graphics/libfive/Portfile

```diff
@@ -5,20 +5,13 @@ PortGroup           github 1.0
 PortGroup           cmake 1.1
 PortGroup           boost 1.0
 
-# Version 601730dc8181f12aeb0c6c66bbe3f4f9b4523134 below does not build
-# with Eigen 3.4, which is the current Eigen version in MacPorts. Newer
-# versions require macOS 10.15 or later to be able to build with
-# Eigen 3.4+, so the update should be done by someone with access 10.15+.
-# See https://trac.macports.org/ticket/63664
-
 # libfive does not have releases; the master branch is recommended.
 # See https://libfive.com/download/
-github.setup        libfive libfive 601730dc8181f12aeb0c6c66bbe3f4f9b4523134
-version             20210717
+github.setup        libfive libfive 71899313d36ce14de6646ef760fa6bbc5c0cc067
+version             20241103
 revision            0
 
 categories          graphics math cad
-platforms           darwin
```

Comment:
if you know it will only work on certain systems, please indicate that here. 
In other words: 

platforms           {darwin >= 12}

where you put the correct number of course. Check the guide for guidance on the "platforms" syntax.

Reply:
I am not certain. I propose first merging this and seeing which buildbots fail.